Cargo.toml

```diff
@@ -29,4 +29,4 @@ serde_json = "1.0.60"
 uuid = {version = "0.8", features  = ["serde", "v4"]}
 wascap = "0.6.0"
 wasmbus-rpc = "0.6.0"
-wasmcloud-interface-lattice-control = "0.3.1"
+wasmcloud-interface-lattice-control = {path="../interfaces/lattice-control/rust"}
```

Comment:
This LGTM, only request (of course) is just to change the version to `0.4.0` when it's published